app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -225,7 +221,7 @@ class SendAdaPage extends Component<Props> {
           />
         )}
         {rawTransactionOpen && <RawTransactionModal />}
-        {showConfirmTransactionDialog && <ConfirmTransactionDialog total={total} />}
+        {showConfirmTransactionDialog && <ConfirmTransactionDialog />}
```

Comment:
why recalculate total in ConfirmTransactionDialog?

Reply:
Because it should be it's responsibility to correctly calculate the value, especially given the fact that we do not supply any other prop to it.